pkg/constants/constants.go

```diff
@@ -115,6 +115,12 @@ const (
 	Explain InferenceServiceVerb = "explain"
 )
 
+// Custom URLS Annotation
+const (
+	CustomURLAnnotation InferenceServiceVerb = "custom.urls"
```

Comment:
Can you move the annotation here https://github.com/kubeflow/kfserving/blob/e3e176b58167ca8b9897a95a35eb913589c41ca6/pkg/constants/constants.go#L48 and prefix with kfserving group name

Reply:
what do you think `KFServingAPIGroupName + "/custom-predictor-urlpaths"` ?

Reply:
accept